main/src/ca/uwaterloo/flix/language/phase/Kinder.scala

```diff
@@ -171,9 +171,9 @@ object Kinder {
         case (sigSym, sig0) => visitSig(sig0, kenv, taenv, root).map(sig => sigSym -> sig)
       }
       val lawsVal = traverse(laws0)(visitDef(_, kenv, taenv, root))
-
-      mapN(tparamVal, superClassesVal, sigsVal, lawsVal) {
-        case (tparam, superClasses, sigs, laws) => KindedAst.Class(doc, mod, sym, tparam, superClasses, sigs.toMap, laws, loc)
+      val annVal = traverse(ann0)(visitAnnotation(_, kenv, taenv, root))
```

Comment:
Nitpick: Can you move this to above `tparamVal` so we maintain the order?

Reply:
sure!